src/wp-includes/class-wp-dependency.php

```diff
@@ -59,6 +59,14 @@ class _WP_Dependency {
 	 */
 	public $args = null;  // Custom property, such as $in_footer or $media.
```

Comment:
Adding an argument after a potentially null mixed $args would not work well. What happenns when there's no in footer or media parameter? It can get mixed into the $args bag.